mod-invoice-storage/schemas/adjustment.json

```diff
@@ -3,26 +3,65 @@
   "description": "Adjustment",
   "type": "object",
   "properties": {
-    "adjustmentLabel": {
+    "id": {
```

Comment:
According to [MODINVOICE-54](https://issues.folio.org/browse/MODINVOICE-54):
> *NOTE* The invoiceLine adjustments maintain a link back to the invoice adjustment. This is needed in order to allow for the invoice adjustment to be modified/removed and have those changes trickle down into the invoiceLine adjustments.

Looking at the samples there I see `id` in the invoice level adjustment:
```json
{
  "id":"14263aab-b22b-4ddc-9ecc-3434427c2c8f",
  "adjustmentLabel":"Service Fee",
  "adjustmentType":"Amount",
  "adjustmentValue":10,
  "prorate":"By line"
}
```
`adjustmentId` in the invoice line level adjustment
```json
{
  "adjustmentId":"14263aab-b22b-4ddc-9ecc-3434427c2c8f",
  "adjustmentLabel":"Service Fee",
  "adjustmentType":"Amount",
  "adjustmentValue":5,
  "prorate":"By line"
}
```
@craigmcnally is this schema change going to be made in separate story?

Reply:
I missed this. We might as well make the change now.

Reply:
Why can't they have the same field name "id" , it can still refer to the invoice level adjustment

Reply:
I think it's confusing to have two adjustments (in invoice and invoice-line) with the same value in "id".  Having a separate "adjustmentsId" makes it clearer (at least to me) that we're referencing another adjustment record.  "id" is typically used to say this is the identifier for *this* record, not some other record.